frontend/tests/ui/util.py

```diff
@@ -231,16 +232,18 @@ def add_identities_course(driver, course_alias):
              '//div[contains(concat(" ", normalize-space(@class), " "), '
              '" upload-csv ")]/div/a'))).click()
 
-    usernames_webdriver_elements = driver.browser.find_elements_by_xpath(
-        '//td[contains(concat(" ", normalize-space(@class), " "), " username "'
-        ')]/strong')
-    passwords_webdriver_elements = driver.browser.find_elements_by_xpath(
-        '//td[contains(concat(" ", normalize-space(@class), " "), " password "'
-        ')]')
-    usernames = [username.text for username in usernames_webdriver_elements]
-    passwords = [password.text for password in passwords_webdriver_elements]
-
-    identities = zip(usernames, passwords)
+    username_elements = driver.browser.find_elements_by_xpath(
+        '//table[contains(concat(" ", normalize-space(@class), " "), " '
+        'identities-table ")]/tbody/tr/td[contains(concat(" ", '
+        'normalize-space(@class), " "), " username ")]/strong')
+    password_elements = driver.browser.find_elements_by_xpath(
+        '//table[contains(concat(" ", normalize-space(@class), " "), " '
+        'identities-table ")]/tbody/tr/td[contains(concat(" ", '
+        'normalize-space(@class), " "), " password ")]')
+    usernames = [username.text for username in username_elements]
+    passwords = [password.text for password in password_elements]
+    Identity = collections.namedtuple('Identity', ['username', 'password'])
```

Comment:
no usaste el NamedTuple de `typing` como decía el link que te dejé :(

```suggestion
from typing import NamedTuple, Text

Identity = typing.NamedTuple('Identity', [('username', Text), ('password', Text)])
```

y mueve `Identity` a L33 porque es la definición de un tipo nuevo

Reply:
No logro entender como funciona. Si lo agrego así:
```
class Identity(NamedTuple):
    username: str
    password: str
```
Me dice que `NamedTuple` no está definido. ¿Qué es lo que estoy haciendo mal?

Reply:
hiciste el `from typing import NamedTuple, Text`?

Reply:
No, ahorita intento con ese cambio

Reply:
Listo, creo que ahora si quedó